packages/generate-arg-types/src/cli.ts

```diff
@@ -14,7 +15,24 @@ const options = {
   shouldRemoveUndefinedFromOptional: true,
 };
 
-const componentsGlobString = process.argv.slice(2).pop();
+const CLI_ARGS_OPTIONS = {
+  allowPositionals: true,
+  options: {
+    exclude: {
+      type: "string",
+      multiple: true,
+      short: "e",
+    },
+  },
+  strict: true,
+} as const satisfies ParseArgsConfig;
+
+const cliArgs = parseArgs({ args: process.argv.slice(2), ...CLI_ARGS_OPTIONS });
+
+console.log(cliArgs);
+
+const componentsGlobString = cliArgs.positionals.join(" ");
```

Comment:
`const globs = cliArgs.positionals`?

Reply:
They are inside '' to not allow system expand, so same logic as before

Reply:
Right after this there is `componentsGlobString.split(' ')`. What system expand?

Reply:
system expands pathes not inside ''

Reply:
`echo */*` is not same as `echo '*/*'`

Reply:
https://github.com/webstudio-is/webstudio-builder/blob/ab903af1303180f178a6d464709742d7381539e6/packages/sdk-components-react-radix/package.json#L44

Reply:
Got it. Could be a one liner with comment `const globs = cliArgs.positionals.join(" ").split(" ")`